scribus/ui/propertywidget_opentypefontfeatures.cpp

```diff
@@ -43,16 +43,70 @@ void PropertyWidget_OpenTypeFontFeatures::showFontFeatures(QString s)
 {
 	if (!m_ScMW || m_ScMW->scriptIsRunning())
 		return;
-	fontfeatures->setText(s);
```

Comment:
If the code works fine (font features are saved/read from file and get applied to the current text frame as expected) then just remove the unused code, otherwise find where it should be used and use it.